README.md

```diff
@@ -12,6 +12,7 @@ the docker images used to build the various GCC cross-compilers used on the site
 - check it builds correctly:
   `./ct-ng build`
 - copy the config in this repository in `build/latest` following the naming convention.
+- build the container as a one-off with `docker build -t gcc-cross .`
```

Comment:
Oh, of course. Sorry I forgot that line :sweat_smile:

Reply:
It took me a bit to realise!